client/post-editor/media-modal/detail/detail-fields.jsx

```diff
@@ -25,20 +25,30 @@ class EditorMediaModalDetailFields extends Component {
 	static propTypes = {
 		site: PropTypes.object,
 		item: PropTypes.object,
+		onUpdate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		onUpdate: noop,
 	};
 
 	constructor() {
 		super( ...arguments );
-		this.persistChange = debounce( this.persistChange, 1000 );
+		this.persistChange = debounce( this._persistChange, 1000 );
```

Comment:
When it comes to deciding the debouncing interval, I personally choose a value that is around 300ms. This is considering that average human typing CPM is about 200. So 1000ms / (200chars / 60 secs) = 300ms.

Reply:
I just respect the original value to keep my changes as small as possible even though the number `1000` looks like a random magic number. :)